admin/templates/configuration/etc/init.d/qserv-czar

```diff
@@ -46,6 +49,17 @@ export QSW_RESULTDIR=${QSERV_RUN_DIR}/qserv-run/tmp
 # Disabling buffering in python in order to enable "real-time" logging.
 export PYTHONUNBUFFERED=1
 
+export XRD_LOGLEVEL=Debug
+export XRDSSIDEBUG=1
+
+# Increase timeouts, without that, long-running queries fail.
+# The proper fix is coming, see DM-3433
+export XRD_REQUESTTIMEOUT=64000
+export XRD_STREAMTIMEOUT=64000
+export XRD_DATASERVERTTL=64000
+export XRD_TIMEOUTRESOLUTION=64000
+
+
```

Comment:
We should put these 'configuration' parameters in QSERV_RUN_DIR/etc/sysconfig/xrootd and source it, it would be easier to maintain than in a startup script, which shouldn't contains configuration /param/values. Same think for PYTHONUNBUFFERED.